build/template-desktop-unit-and-automation.yaml

```diff
@@ -1,18 +1,8 @@
 steps:
 - task: UseDotNet@2
-  displayName: 'Use .Net Core SDK 3.x'
+  displayName: 'Use the latest .NET 8'
   inputs:
-    version: 3.x
-
-- task: UseDotNet@2
-  displayName: 'Use .Net Core SDK 5.x'
-  inputs:
-    version: 5.x    
-
-- task: UseDotNet@2
-  displayName: 'Use .Net Core SDK 6.x'
-  inputs:
-    version: 6.x
+    version: 8.x
     includePreviewVersions: true
 
```

Comment:
You probably don't need this anymore

Reply:
Yea, that was probably for when Maui first came out.